osu.Game.Tests/Visual/Multiplayer/TestSceneMultiSpectatorScreen.cs

```diff
@@ -333,10 +354,31 @@ private void start(int[] userIds, int? beatmapId = null)
             {
                 foreach (int id in userIds)
                 {
-                    OnlinePlayDependencies.Client.AddUser(new User { Id = id }, true);
+                    var user = new MultiplayerRoomUser(id)
+                    {
+                        User = new User { Id = id },
+                    };
 
+                    OnlinePlayDependencies.Client.AddUser(user.User, true);
                     SpectatorClient.StartPlay(id, beatmapId ?? importedBeatmapId);
-                    playingUsers.Add(new MultiplayerRoomUser(id));
+
+                    playingUsers.Add(user);
+                }
+            });
+        }
+
+        private void end(int[] userIds)
```

Comment:
```suggestion
        private void end(int userId)
```

Looks like the array is never being used.